src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_TypeInference.cs

```diff
@@ -1533,6 +1533,7 @@ static void M()
                 Diagnostic(ErrorCode.WRN_ConvertingNullableToNonNullable, "null").WithLocation(14, 13));
         }
 
+        // PROTOTYPE(NullableReferenceTypes): Deconstruction declaration ignores nullability.
```

Comment:
>Deconstruction declaration  [](start = 46, length = 27)

It doesn't look like this unit-test uses deconstruction declaration. Did you mean to say "deconstruction assignment"?